test/test_demand_api.py

```diff
@@ -28,117 +27,262 @@ def setUp(self):
     def tearDown(self):
         pass
 
-    def test_demand_actual_total_get(self):
+    @patch('elexonpy.api.demand_api.DemandApi.demand_actual_total_get')
```

Comment:
This feels a bit to high up to mock. I wonder if you could mock that acually api respsnse?

Reply:
Ive seen things like this used before - https://stackoverflow.com/questions/15753390/how-can-i-mock-requests-and-the-response